docs/docsite/rst/user_guide/intro_inventory.rst

```diff
@@ -478,6 +478,96 @@ Here is an example of how to instantly deploy to created containers::
 .. note:: If you're reading the docs from the beginning, this may be the first example you've seen of an Ansible playbook. This is not an inventory file.
           Playbooks will be covered in great detail later in the docs.
 
+.. _using_multiple_inventory_sources:
+
+Using multiple inventory sources
```

Comment:
What is the use case here? Why would I want to structure my static inventory this way? What problem does this structure solve? If there's a specific scenario, that makes the example easier to follow.

Reply:
so you can have specific inventories for different things, but then combine them when you need 'global access' i.e `-i /path/to/development -i /otherpath/to/production`